sync/sync_head.go

```diff
@@ -172,7 +172,12 @@ func (s *Syncer[H]) verify(ctx context.Context, newHead H) (bool, error) {
 
 	var verErr *header.VerifyError
 	if errors.As(err, &verErr) && !verErr.SoftFailure {
-		log.Errorw("invalid network header",
+		logF := log.Warnw
+		if errors.Is(err, header.ErrKnownHeader) {
+			logF = log.Debugw
+			log.Level()
```

Comment:
why is this necessary?

Reply:
it is not there. I think it is some GH issue, as file shows outdated